appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jmac/ConfigDomainParser.java

```diff
@@ -41,21 +33,34 @@
 import org.glassfish.api.admin.ServerEnvironment;
 import org.glassfish.internal.api.Globals;
 import org.jvnet.hk2.config.types.Property;
+import org.omnifaces.eleos.config.factory.ConfigParser;
+import org.omnifaces.eleos.data.AuthModuleConfig;
+import org.omnifaces.eleos.data.AuthModulesLayerConfig;
 
-import static java.util.regex.Matcher.quoteReplacement;
+import com.sun.enterprise.config.serverbeans.MessageSecurityConfig;
+import com.sun.enterprise.config.serverbeans.ProviderConfig;
+import com.sun.enterprise.config.serverbeans.RequestPolicy;
+import com.sun.enterprise.config.serverbeans.ResponsePolicy;
+import com.sun.enterprise.config.serverbeans.SecurityService;
+import com.sun.logging.LogDomains;
+
+import jakarta.security.auth.message.MessagePolicy;
 
 /**
  * Parser for message-security-config in domain.xml
  */
 public class ConfigDomainParser implements ConfigParser {
 
-    private static final Logger _logger = LogDomains.getLogger(ConfigDomainParser.class, LogDomains.SECURITY_LOGGER);
+    private static Logger _logger = null;
```

Comment:
Looks like a revert of recent improvements in master branch. it's better to initialize the field directly and not in a separate static block.